flytekit/core/task.py

```diff
@@ -69,58 +69,62 @@ def task(
     **kwargs,
 ) -> Union[Callable, PythonFunctionTask]:
     """
-    This is the core decorator to use for any task type in FlyteKit.
-    Usage: for a simple python task
+    This is the core decorator to use for any task type in flytekit. For a simple python task,
```

Comment:
did you mean to add more at the end here?

Reply:
no, usage follows, but I'm using a code block, so I can't use ::